scripts/generate_shields.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # Generate highway shields as SVG files in symbols/shields.
 
```

Comment:
Line 5 says:

    from __future__ import print_function

but this can be removed now. In other files as well.